.github/workflows/cifuzz.yml

```diff
@@ -7,10 +7,14 @@ on:
     branches:
       - master
       - branch/*
+    paths:
+      - '**.go'
```

Comment:
Maybe also skip any .go files under .github/ as we don't implement any fuzzing tests for the GitHub bot.

Reply:
You can't use both `paths` and `paths-ignore` in the same workflow, unfortunately.